pack/side/wog_encounter.json

```diff
@@ -0,0 +1,1035 @@
+        "illustrator": "Joshua Cairos",
+        "is_unique": true,
+        "name": "Silenus",
+        "pack_code": "wog",
+        "position": 35,
+        "quantity": 1,
+        "subname": "The Empty Sky",
+        "text": "*Silenus has a global health pool of 8[per_investigator].\nHunter. Massive.\n<b>Forced</b> - When the mythos phase ends: Each investigator at Silenus's location must decide (choose one) -\n- Place 1 doom on the blue agenda.\n- Remove each card in your hand from the game.\n- Take 3 horror.",
+        "traits": "Ancient One. Elite.",
+        "type_code": "enemy",
+        "victory": 2
+    },
+    {
+        "code": "86036",
+        "encounter_code": "death_of_stars",
+        "encounter_position": 3,
+        "enemy_damage": 1,
+        "enemy_evade": 5,
+        "enemy_fight": 3,
+        "enemy_horror": 2,
+        "faction_code": "mythos",
+        "health": 8,
+        "illustrator": "Matthew Cowdery",
+        "is_unique": true,
+        "name": "The Inescapable Maw",
+        "pack_code": "wog",
+        "position": 36,
+        "quantity": 1,
+        "subname": "Servant of Silenus",
+        "text": "Hunter.\nAnytime an investigator is forced to \"decide\" by an ability on a blue encounter card, they must choose two options instead of one.",
+        "traits": "Humanoid. Servitor. Elite.",
+        "type_code": "enemy",
+        "victory": 1
+    },
+    {
+        "code": "86037",
+        "encounter_code": "death_of_stars",
+        "encounter_position": 4,
+        "enemy_damage": 1,
+        "enemy_evade": 2,
+        "enemy_fight": 3,
+        "enemy_horror": 2,
+        "faction_code": "mythos",
+        "health": 4,
+        "illustrator": "Matthew Cowdery",
+        "name": "Hune-Stitched Herald",
+        "pack_code": "wog",
+        "position": 37,
+        "quantity": 1,
+        "text": "Warring.\n<b>Forced</b> - After Hune-Stitched Herald engages you: You must decide (choose one) -\n- Place 1 doom on the blue agenda.\n- Remove 2 clues from around Hub Dimension.\n- Hune-Stitched Herald attacks you.",
+        "traits": "Monster.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "86038",
+        "encounter_code": "death_of_stars",
+        "encounter_position": 5,
+        "enemy_evade": 1,
+        "enemy_fight": 2,
+        "enemy_horror": 1,
+        "faction_code": "mythos",
+        "health": 3,
+        "illustrator": "Adam Doyle",
+        "name": "Ethereal Entity",
+        "pack_code": "wog",
+        "position": 38,
+        "quantity": 3,
+        "text": "Warring.\n<b>Spawn</b> - Athenaeum of the Empty Sky.\n<b>Forced</b> - After Ethereal Entity engages you: You must decide (choose one) -\n- Place 1 doom on the blue agenda.\n- Place 1 of your clues on your location.\n- Take 2 horror.",
+        "traits": "Monster.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "86039",
+        "encounter_code": "death_of_stars",
+        "encounter_position": 7,
```

Comment:
Position 8 please (changing the positions to make them reflect the quantities of the additional enemy).